dvc/remote/local/__init__.py

```diff
@@ -122,6 +123,7 @@ def makedirs(self, path_info):
         if not self.exists(path_info):
             os.makedirs(path_info["path"])
 
+    @slow_link_aware
```

Comment:
So is this warning going to be printed for each file that we link?

Reply:
hmm, that is right, not very clever